client/components/data/query-sites/README.md

```diff
@@ -0,0 +1,8 @@
+Query Sites
+===========================
+
+`<QuerySites />` is a React component used in managing network requests for sites.
```

Comment:
If you pass an ID to this query it could very well hit `site/:site` now or in the future.